code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -552,42 +644,27 @@
 	cyborg_base_icon = "service_m" // display as butlerborg for radial model selection
 	special_light_key = "service"
 	hat_offset = 0
+	borg_skins = list(
+		"Waitress" = list(SKIN_ICON_STATE = "service_f", SKIN_TRANSFORM = TRUE),
+		"Butler" = list(SKIN_ICON_STATE = "service_m", SKIN_TRANSFORM = TRUE),
+		"Bro" = list(SKIN_ICON_STATE = "brobot", SKIN_TRANSFORM = TRUE),
+		"Kent" = list(SKIN_ICON_STATE = "kent", SKIN_LIGHT_KEY = "medical", SKIN_HAT_OFFSET = 3, SKIN_TRANSFORM = TRUE),
+		"Tophat" = list(SKIN_ICON_STATE = "tophat", SKIN_LIGHT_KEY = null, SKIN_HAT_OFFSET = INFINITY, SKIN_TRANSFORM = TRUE),
```

Comment:
by your own logic making `SKIN_LIGHT_KEY` null won't do anything

Reply:
Just an oversight, I've fixed it up.